esignet-service/src/main/resources/application-local.properties

```diff
@@ -254,14 +254,18 @@ crypto.PrependThumbprint.enable=true
 mosip.esignet.ui.wallet.config={{'wallet.name': 'Inji', 'wallet.logo-url': 'inji_logo.png', 'wallet.download-uri': '#', \
  'wallet.deep-link-uri': 'inji://landing-page-name?linkCode=LINK_CODE&linkExpireDateTime=LINK_EXPIRE_DT' }}
 
+knowledge.field.details={{'id':'policyNumber', 'type':'text', 'format':''},{'id':'name', 'type':'text', 'format':''},{'id':'dob', 'type':'date', 'format':'dd/mm/yyyy'}}
```

Comment:
rename the key to mosip.esignet.authenticator.default.auth-factor.kba.field-details

Reply:
renamed to mosip.esignet.authenticator.default.auth-factor.kba.field-details